calculadora_do_cidadao/adapters/cpi.py

```diff
@@ -0,0 +1,81 @@
+from datetime import date
+from decimal import Decimal
+from typing import NamedTuple
+from urllib.parse import urlencode
+
+from rows.fields import DecimalField, DateField
+
+from calculadora_do_cidadao.adapters import Adapter
+from calculadora_do_cidadao.months import MONTHS
+from calculadora_do_cidadao.typing import MaybeIndexesGenerator
+
+
+today_date = str(date.today())
+
```

Comment:
Do you think this blank line substantially helps in terms of readability?

Reply:
Here, not at all! It messes me around actually LOL.